md/terminusx/start-with-a-client.md

```diff
@@ -2,7 +2,7 @@
 
 > **On this page:** A step-by-step guide with examples to install and get started with a JavaScript or Python client.
 
-This guide demonstrates the basic use of the **WOQLClient** library to connect to TerminusX with a JavaScript or Python client. Refer to [TerminusDB JavaScript Client](https://terminusdb.github.io/terminusdb-client-js/) or [TerminusDB Python Client](https://terminusdb.github.io/terminusdb-client-python/) for detailed documentation. If you want to access the code which is discussed in this page [here](../../code-examples/start-with-client/) are the Javascript and Python code examples.
+This guide demonstrates the basic use of the **WOQLClient** library to connect to TerminusX with a JavaScript or Python client. Refer to [TerminusDB JavaScript Client](https://terminusdb.github.io/terminusdb-client-js/) or [TerminusDB Python Client](https://terminusdb.github.io/terminusdb-client-python/) for detailed documentation. If you want to access the code which is discussed in this page [here](https://github.com/terminusdb/terminusdb-docs/tree/main/code-examples/start-with-client) are the Javascript and Python code examples.
 

```

Comment:
I see that the existing link is broken, but the new link, `https://github.com/terminusdb/terminusdb-docs/tree/main/code-examples/start-with-client`, presents us with a problem when the docs branch is not `main` (e.g. it might be `v10.0`, `v10.1`, etc.). We'd rather stick with the same branch as the one shown.

I think we should try using the [`:ignore'` feature](https://docsify.js.org/#/helpers?id=ignore-to-compile-link) for this link.

For now, just revert this particular change from the PR.

Reply:
Done 👍